src/views/Login.vue

```diff
@@ -84,8 +84,6 @@
             login: this.email,
             password: this.password,
           });
-          this.$store.dispatch("fetchAccounts");
```

Comment:
Shouldn't it be removed also in autologin ?

Reply:
I don't know. Why would it be ? Is it useless also ?

Reply:
I tried without, and autologin still works. So in the name of completeness I pushed the change.

It surprises me as it is required in the login process flow to retrieve the money accounts. There's something I don't get (I'll gradly benefit from your lights on this point).

Thanks

Reply:
In your first version of this commit, it seemed that your intention was to have fetchAccount + fetchTransaction in both store's actions login and auto-login, preventing the need to call those through dispatch in the components. Which seems to increase consistency IMHO, and is a welcome change. But it seemed you removed the dispatch of these actions only in the method of `Login.vue` managing the login part, leaving the auto-login code calling a dispatch of `fetchAccounts` in `Core.vue` that seemed now unnecessary.
It seems now, in your last version of the commit, you don't want anymore to do your `fetchAccount` in autologin anymore. Is that intended ? it seemed to my, at first hand, that it was a good idea code-wise.

Reply:
Ahh now I see. Thanks. Overlook from my part. I just forgot it was called from Core.vue.
Yes I wanted all these calls to be in the store. That was intended. But since I misunderstood your comment, I removed the call from the store (and now I understand why it was'nt needed to function properly).